meow/frontend/src/components/EditPost/index.js

```diff
@@ -27,7 +26,9 @@ const contentStyles = { position: "relative", transform: "translateY(-30px)" };
 class EditPost extends React.Component {
   state = {
     sections: this.props.sections,
-    sectionerror: ""
+    sectionerror: "",
+    meowWithIn15Mins: false,
```

Comment:
Nitpick: a bit of a capitalization error with this variable. It should be meowWithin15Mins. The best would be to call it hasMeowWithin15Mins just to make it more clear that it's a bool